core/src/banking_stage.rs

```diff
@@ -558,21 +558,22 @@ impl BankingStage {
 
             let mut measure = Measure::start("banking_stage-forward-us");
 
-            let conn = if let ForwardOption::ForwardTpuVote = forward_option {
+            let res = if let ForwardOption::ForwardTpuVote = forward_option {
                 // The vote must be forwarded using only UDP. Let's get the UDP connection.
                 banking_stage_stats
                     .forwarded_vote_count
                     .fetch_add(packet_vec_len, Ordering::Relaxed);
-                Arc::new(UdpTpuConnection::new_from_addr(addr).into())
+                let conn = UdpTpuConnection::new_from_addr(addr);
+                conn.send_wire_transaction_batch_async(packet_vec)
             } else {
                 // All other transactions can be forwarded using QUIC, get_connection() will use
                 // system wide setting to pick the correct connection object.
                 banking_stage_stats
                     .forwarded_transaction_count
                     .fetch_add(packet_vec_len, Ordering::Relaxed);
-                get_connection(&addr)
+                let conn = get_connection(&addr);
+                conn.send_wire_transaction_batch_async(packet_vec)
```

Comment:
is there any change here?

Reply:
Good catch -- I no longer need this change in this file -- after I removed the reference counting changes in the connection pool entry. I will revert it.

Reply:
done